run.py

```diff
@@ -1,80 +1,115 @@
+        os.symlink(filepath, sanitized_filepath)
+
+    return sanitized_filepath
 
 
 def fail_check(context):
-    
+
     input_path = context.get_input_path("Input_file")
     expected_extension = ".svs"
-    
+
     try:
-        passes = file.is_valid(input_path)#, expected_extension)
+        passes = file.is_valid(input_path)  # , expected_extension)
     except FileNotFoundError as e:
         context.log.exception(e)
-        raise(e)
+        raise (e)
     except TypeError as e:
         context.log.exception(e)
-        raise(e)
+        raise (e)
     except Exception as e:
-        context.log.warning(f'Problem with {input_path}')
+        context.log.warning(f"Problem with {input_path}")
         context.log.exception(e)
-        raise(e)
-    
-    return(passes)
-        
+        raise (e)
+
+    return passes
+
 
 def setup(context):
-    
+
     input_keys = context.config.keys()
-    
+
     # Create the head of the command
-    command = ['wsi2dcm',
-               '--input', context.get_input_path("Input_file"),
+    command = [
+        "wsi2dcm",
+        "--input",
+        create_sanitized_filepath(context.get_input_path("Input_file")),
     ]
-    
+
     # If present add the other input (the dicom json file)
-    if context.get_input('jsonFile'):
-        command.extend(['--jsonFile', context.get_input_path("jsonFile")])
-    
+    if context.get_input("jsonFile"):
+        command.extend(
+            [
+                "--jsonFile",
+                create_sanitized_filepath(context.get_input_path("jsonFile")),
+            ]
+        )
+
     # Append any specified config options
     for key in input_keys:
         if context.config.get(key):
             command.extend([f"--{key}", context.config[key]])
-    
+
     # Give the series description some name if not specified
-    if 'seriesDescription' not in input_keys:
+    if "seriesDescription" not in input_keys:
 
         series_description = Path(context.get_input_path("Input_file")).stem
-        
-        command.extend(['--seriesDescription', series_description])
+
+        command.extend(["--seriesDescription", series_description])
     else:
-        series_description = context.config.get('seriesDescription')
-    
-    output_dir = context.work_dir/Path(series_description)
-    
+        series_description = context.config.get("seriesDescription")
+
+    output_dir = context.work_dir / Path(
+        sanitize_filepath(series_description, platform="auto").replace(" ", "_")
+    )
+
     if not os.path.exists(output_dir):
         os.mkdir(output_dir)
-        
-    command.extend(['--outFolder', output_dir])
+
+    command.extend(["--outFolder", output_dir])
     # Stringify all PATHS in the command for logging stuff.
     command = [str(c) for c in command]
-    
+
     context.log.info("Command to call: \n" + " ".join(command))
-    
-    return(command, series_description, output_dir)
 
+    return (command, series_description, output_dir)
```

Comment:
It's everywhere.  I'm sorry.  I recommend a search and replace for `raise (` -> `raise(` and `return (` -> `return(`